Source/ThirdParty/TurboBadger/tb_widgets_reader.cpp

```diff
@@ -42,6 +42,8 @@ void TBWidget::OnInflate(const INFLATE_INFO &info)
 
     SetOpacity(info.node->GetValueFloat("opacity", GetOpacity()));
 
+    SetDisabledOpacity(info.node->GetValueFloat("disabledOpacity", GetDisabledOpacity()));
```

Comment:
Nice add, one note "disabled-opacity" would be the TB convention for the attr name